web/src/features/panels/ranking-panel/getRankingPanelData.ts

```diff
@@ -33,15 +33,19 @@ export const getRankedState = (
   if (!data) {
     return [];
   }
-  const zonesData = data.data;
-  const keys = Object.keys(zonesData.zones) as Array<keyof GridState>;
+  const girdState = data.data.datetimes[datetimeIndex];
```

Comment:
`gird` should probably be grid here too, right?

Reply:
That's what I get for trying to spell in the morning 😅

I'll take care of it...